README.md

```diff
@@ -1,2 +1,70 @@
-# Math-Magician
-Math magicians is a Single Page App (SPA), based on react,  that allows users to make simple calculations and read random math-related quotes. Built mainly with REACTJS
+# Getting Started with Create React App
+
+This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
+
+## Available Scripts
+
+In the project directory, you can run:
+
+### `npm start`
+
+Runs the app in the development mode.\
+Open [http://localhost:3000](http://localhost:3000) to view it in your browser.
+
+The page will reload when you make changes.\
+You may also see any lint errors in the console.
+
+### `npm test`
+
+Launches the test runner in the interactive watch mode.\
+See the section about [running tests](https://facebook.github.io/create-react-app/docs/running-tests) for more information.
+
+### `npm run build`
+
+Builds the app for production to the `build` folder.\
+It correctly bundles React in production mode and optimizes the build for the best performance.
```

Comment:
- [ ] Nice job on creating the react app but you forgot to add the relevant details to your readme file, please do so and ask for another review.